.github/workflows/push.yml

```diff
@@ -30,4 +30,5 @@ jobs:
       uses: mikeal/merge-release@master
       env:
         GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        NPM_PRIVATE: false
```

Comment:
Let’s leave this out so that we’re testing the default behavior in our own publishes.